src/app/clusters/group-key-mgmt-server/group-key-mgmt-server.cpp

```diff
@@ -119,7 +120,8 @@ class GroupKeyManagementAttributeAccess : public AttributeAccessInterface
         switch (aPath.mAttributeId)
         {
         case GroupKeyManagement::Attributes::ClusterRevision::Id:
-            return ReadClusterRevision(aPath.mEndpointId, aEncoder);
+            return aEncoder
+            ReadClusterRevision(aPath.mEndpointId, aEncoder.encode(kImplementedClusterRevision));
```

Comment:
This part looks like something went awry here.  Why are we adding a new constant instead of changing the existing one?  And this bit of code clearly can't compile.

Reply:
Something went wrong just before the push after I added revision. Will fix